packages/actor-query-operation-orderby-sparqlee/lib/ActorQueryOperationOrderBySparqlee.ts

```diff
@@ -41,10 +41,11 @@ export class ActorQueryOperationOrderBySparqlee extends ActorQueryOperationTyped
     const sparqleeConfig = { ...ActorQueryOperation.getExpressionContext(context) };
     let bindingsStream = output.bindingsStream;
 
-    for (let expr of pattern.expressions) {
+    //sorting backwards since the first one is the most important therefore should be ordered last.
+    for(let i = pattern.expressions.length-1; i >= 0 ;i--){
```

Comment:
I'm surprised our linter does not complain about this, but we typically write our statements with spaces, like this:
```suggestion
    for (let i = pattern.expressions.length-1; i >= 0 ;i--) {
```
(also applies to if-statements and so on)